### DISCUSSION TITLE
@probonopd Are you aware of this and is this ok?

### DISCUSSION THREAD

**Discussion by DutchBigEd** [probonopd/MiniDexed#667] [General]
![IMG-20240621-WA0002](https://github.com/probonopd/MiniDexed/assets/17274454/fe3f8d5b-a075-4bb2-9bc6-7dc82748fccd)
Translation:
Eight in one 
The Dutch company D-tronics makes handy programmers for vintage synths, but they're now working on an ultra-compact 8-fold DX7, which will be called DT-DX.
The module has been loaned to Probonopd's MiniDexed, one on one Raspbery Pi based DX7. 
D-Tronics uses a larger Raspbery and made a nice housing, where by something similar to the 8-fold TX816 and TX802 from Yamaha, but in a super compact box with a rotary knob and display, compressor and Reverb, plus MIDI and USB. 
This DT-DX can load syx files and if you really want to edit sounds, you can of course do so with D-Tronics own spacious DT-7 hardware programmer. 
For the time being, It's a Kickstarter project and there's no price yet.

**Comment by BobanSpasic** [comment]
https://github.com/probonopd/MiniDexed/discussions/637

**Comment by probonopd** [comment]
Yes, anyone is free to make MiniDexed compatible hardware and sell it. As long as you point to https://github.com/probonopd/MiniDexed and follow its license.
